js/functions.js

```diff
@@ -0,0 +1,78 @@
+/* Домашнее задание
+Функция №1 для проверки длины строки (2 варианта). */
+
+function checkLength1 (row, maxLen) {
```

Comment:
```suggestion
function checkLength1 (row, maxLength) {
```

Нарушаем [критерий](https://up.htmlacademy.ru/javascript/28/criteries#b4) - давай уберем все сокращения, которые не разрешены нам по правилам.